lib/logger.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
```

Comment:
I prefer to use `debug` library to log debug messages.

Reply:
I can try it for the next release.